OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -110,13 +113,21 @@ public override Activity Tick(Actor self)
 				else
 					return new FlyCircle(self);
 			}
+			//// ignore destination generated by Calculate() if they are too close to self
```

Comment:
Please add a new line in front of the comment and start it with only two slashes and a capital:
`// Ignore destintation...`